src/l200geom/materials/surfaces.py

```diff
@@ -107,6 +110,30 @@ def to_sipm_silicon(self) -> g4.solid.OpticalSurface:
 
         # add custom efficiency for the KETEK SiPMs. This is not part of legendoptics.
         λ, eff = ketek_sipm_efficiency()
+
+        # Check whether to use the KETEK efficiency curve. If not, use flat 1s.
+        if not getattr(runtime_config, "use_pde_curve", False):
```

Comment:
```suggestion
        if not runtime_config.get("use_pde_curve", False):
```

is a bit more concise (an `AttrsDict` is also just a `dict`, so you can use it like one)